### PR TITLE
chore(plugin): add scope equals to global attribute for some plugins

### PR DESCRIPTION
**Pull request by tzssangglass**
Signed-off-by: tzssangglass <tzssangglass@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/admin/plugins.lua [hunk 1]
```diff
@@ -48,6 +48,7 @@ function _M.get(name)
             metadata_schema = true,
             consumer_schema = true,
             type = true,
+            scope = true,
```
Comment (spacewander) [inline apisix/admin/plugins.lua:51]: Let's update `/v1/schema` too: https://github.com/apache/apisix/blob/master/docs/en/latest/control-api.md#get-v1schema

**Review by spacewander — COMMENTED** [review]
Also need to update plugin node-status.

**Comment by tzssangglass** [comment]
> Also need to update plugin node-status.

The doc for node-status needs to be update: https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/node-status.md#how-to-enable

the example says to enable node-status on route.

File: apisix/admin/plugins.lua (hunk 1 above)
Comment (tzssangglass) [reply apisix/admin/plugins.lua:51]: done

File: t/control/schema.t [hunk 2]
```diff
@@ -98,3 +98,48 @@ __DATA__
...
+                if v.scope == "global" then
+                    global_plugins[k] = v.scope
+                end
+            end
+            ngx.say(json.encode(global_plugins))
+        }
+    }
+--- response_body
+{"batch-requests":"global","error-log-logger":"global","node-status":"global","server-info":"global"}
+--- no_error_log
```
Comment (spacewander) [inline t/control/schema.t:144]: The no_error_log is already set in the file level

Comment (tzssangglass) [reply t/control/schema.t]: done

**Review by spacewander — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**